README.md

```diff
@@ -1,131 +1,171 @@
 
-If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. Edit your server configuration as follows:
+![Screenshot from 2022-03-16 11-47-14](https://user-images.githubusercontent.com/1933730/158573673-723e77a2-6d58-4aa5-8a89-6adcbce50e13.png)
 
-```json
-{
-    "ServiceSettings": {
-        ...
-        "EnableLocalMode": true,
-        "LocalModeSocketLocation": "/var/tmp/mattermost_local.socket"
-    }
-}
-```
+The dialog looks like this:
 
-and then deploy your plugin:
-```
-make deploy
-```
+![Screenshot from 2022-03-16 11-51-05](https://user-images.githubusercontent.com/1933730/158573834-70ea72b0-4a03-4b09-a694-751c0ca1ba04.png)
 
-You may also customize the Unix socket path:
-```
-export MM_LOCALSOCKETPATH=/var/tmp/alternate_local.socket
-make deploy
-```
+- User: The user you want to grant the badge to (may be prepopulated if you clicked the grant button from the profile popover, or added the username in the command).
+- Badge: The badge you want to grant (may be prepopulated if you added the badge id in the command).
+- Reason: An optional reason why you are awarding this badge. (Specially useful for badges like "Thank you")
+- Notify on this channel: If you mark this, a message from the badges bot will be posted in the current channel, letting all know that you granted this badge to that person.
+When a badge is granted, three things may happen:
+- The user that received the badge will always receive a DM from the badges bot letting them know they have been awarded a badge.
+- If "Notify on this channel" was marked, the badges bot will post a message on the current channel letting everyone know that the user has been awarded a badge.
+- If a subscription for this badge type is set, the badges bot will post a message on all subscribed channels letting everyone know that the user has been awarded a badge.
 
-If developing a plugin with a webapp, watch for changes and deploy those automatically:
-```
-export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
-export MM_ADMIN_TOKEN=j44acwd8obn78cdcx7koid4jkr
-make watch
-```
+![Screenshot from 2022-03-16 12-28-45](https://user-images.githubusercontent.com/1933730/158580318-592bb139-6c43-48f0-99c3-79d868aa8024.png)
 
-### Deploying with credentials
+If you try to award a not multiple badge to any user that already has it, nothing will happen (no messages will be created).
 
-Alternatively, you can authenticate with the server's API with credentials:
-```
-export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
-export MM_ADMIN_USERNAME=admin
-export MM_ADMIN_PASSWORD=password
-make deploy
-```
+### Subscriptions
+In order to create a subscription, you must be a badges admin.
+There are two ways to open the subscription creation dialog: `/badges subscription create` and the "Add badge subscription" menu from the channel menu.
```

Comment:
See previous note about potential ways to introduce a step that has more than one way through to an end goal

Also - I'm a bit unclear about subscriptions and how they relate to badges. I feel that subscriptions may be a new concept at this point and may require a bit of explanation or context.